### PR TITLE
Update phone number exercise

### PR DESCRIPTION
**Pull request by djspinmonkey**
# pull request

The main change here is to rename the `.getNumber()` method implied by the tests to be called `.toString()` instead.

A phone number either warrants its own type or it does not. If it does, then what this method is returning is the canonical string representation of a given phone number, and in Java we call methods that do that `.toString()`.

(If a phone number does _not_ warrant its own type, then this exercise should be significantly restructured to use a `PhoneNumberFormatter` class that works with Strings, or some such, but that doesn't really seem necessary and would be a much larger change.)

The other minor changes are to clean up the formatting of the description a bit, and remove a reference to the exercise being "difficulty 5", which to the best of my knowledge does not mean anything in the context of Exercism.

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/main/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/practice/phone-number/.docs/instructions.append.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
 # Instructions append
 
-Since this exercise has difficulty 5 it doesn't come with any starter implementation.
-This is so that you get to practice creating classes and methods which is an important part of programming in Java.
-It does mean that when you first try to run the tests, they won't compile.
-They will give you an error similar to:
+This exercise doesn't come with any starter implementation.  This is so that you get to practice creating classes and
+methods, which is an important part of programming in Java.  That means that when you first try to run the tests, they
+won't compile.  They will give you an error similar to:
```
Comment (SleeplessByte) [inline exercises/practice/phone-number/.docs/instructions.append.md:5]: We adhere to the [Markdown style](https://exercism.org/docs/building/markdown/markdown) given by Exercism, which is one sentence per line. Can you revert these changes in this file?

File: exercises/practice/phone-number/src/test/java/PhoneNumberTest.java [hunk 2]
```diff
@@ -10,7 +10,7 @@ public class PhoneNumberTest {
     @Test
     public void cleansTheNumber() {
         String expectedNumber = "2234567890";
-        String actualNumber = new PhoneNumber("(223) 456-7890").getNumber();
+        String actualNumber = new PhoneNumber("(223) 456-7890").toString();
```
Comment (SleeplessByte) [inline exercises/practice/phone-number/src/test/java/PhoneNumberTest.java:13]: If the tests are changed, the _proof_ or _example_ implementation in the `phone-number` folder also needs to be updated, otherwise the CI will fail to run.

**Comment by sanderploegsma** [comment]
Closing this because of inactivity. @djspinmonkey let us know if you still want to finish this, we’ll reopen this PR.
